### PR TITLE
Fix available validation

### PR DESCRIPTION
**Pull request by kel-man**

### CODE REVIEW

File: app/javascript/components/pages/PetNew.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ class PetNew extends React.Component {
         behavior:"",
         city:"",
         state:"",
-        available:true,
+        available:'',
```
Comment (sjproctor) [inline app/javascript/components/pages/PetNew.js:29]: You should be consistent with your quotation style

File: app/javascript/components/pages/PetNew.js [hunk 2]
```diff
@@ -207,7 +207,7 @@ class PetNew extends React.Component {
                   aria-label="Available Now"
                   />
                   <FormControlLabel
-                  value="false"
+                  value='false'
```
Comment (sjproctor) [inline app/javascript/components/pages/PetNew.js:210]: Same thing here with the quotation marks.

File: app/controllers/pets_controller.rb [hunk 3]
```diff
@@ -11,8 +11,8 @@ def show
   end
 
   def create
-      pet = current_user.pets.create(pet_params)
-      render json: pet
+    pet = current_user.pets.create(pet_params)
+    render json: pet
```
Comment (sjproctor) [inline app/controllers/pets_controller.rb:15]: I would think some error handling here might be a good idea.

**Review by sjproctor — COMMENTED** [review]
Couple notes.

File: app/controllers/pets_controller.rb (hunk 3 above)
Comment (kel-man) [reply app/controllers/pets_controller.rb:15]: Will do

File: app/javascript/components/pages/PetNew.js [hunk 4]
```diff
@@ -51,11 +51,11 @@ class PetNew extends React.Component {
     console.log(this.state)
   }
 
-  handleAvailability = (e) => {
-    const value = e.target.value === "true"
-    let { pet } = this.state
-    this.setState({ pet: { ...pet, available: value } })
-  }
+  // handleAvailability = (e) => {
```
Comment (sjproctor) [inline app/javascript/components/pages/PetNew.js:54]: Is this being used?

Comment (kel-man) [reply app/javascript/components/pages/PetNew.js:54]: Nope, good catch. My bad on that

**Review by sjproctor — APPROVED** [review]
Looks good!

**Pull request merged**